test/external.js

```diff
@@ -8,14 +8,15 @@ const expect = require('expect');
 const root = 'test/'
 
 function sh(cmd) {
-    return child_process.execSync(cmd, {encoding: 'utf-8', stdio: 'pipe', cwd: root});
+    return child_process.spawnSync(cmd, {shell: true, encoding: 'utf-8', stdio: 'pipe', cwd: root})
```

Comment:
What's the reason for using `spawn` instead of `exec`? :-)

Reply:
So we can get back `stderr`. The docs for child_process are a hard to read so I may have missed something, but this definitely works.

Reply:
Okay, great. :)